cmd/kanikoExecute_test.go

```diff
@@ -74,6 +74,10 @@ func (f *kanikoFileMock) Chmod(path string, mode os.FileMode) error {
 	return fmt.Errorf("not implemented. func is only present in order to fullfil the interface contract. Needs to be ajusted in case it gets used.")
 }
 
+func (f *kanikoFileMock) Abs(path string) (string, error) {
+	return "", fmt.Errorf("not implemented. func is only present in order to fullfil the interface contract. Needs to be ajusted in case it gets used.")
+}
+
```

Comment:
Well I think the issue is more that these commands have their own mock implementation but use the common interface.

Reply:
Exactly. I am saying that they should define their specific interface instead of using a "common one". The common can be removed, it seems like it is just a burden and does not align with how it is supposed to work in Go.

Reply:
I mean in the runtime portion, not in the test. They should define their runtime interface, which will be a subset of what Files and FilesMock each implement.

Reply:
Should be improved in a follow up pr, e.g. by removing the duplicated mock implementation